### PR TITLE
Update url to server for quickstart

### PR DESCRIPTION
**Pull request by rancher-max**
#### Proposed Changes ####

Update quickstart doc for agent node join.

#### Types of Changes ####

Docs

#### Linked Issues ####

N/A

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ vim /etc/rancher/rke2/config.yaml
 ```
 Content for config.yaml:
 ```bash
-url: https://<server>:9345
+server: https://<server ip>:9345
```
Comment (briandowns) [inline docs/quickstart.md:65]: Cna a hostname be used here as well?

**Review by cjellick — APPROVED** [review]

File: docs/quickstart.md (hunk 1 above)
Comment (rancher-max) [reply docs/quickstart.md:65]: yes it can! Do you think "server" is more clear then? Or "server address" maybe?

Comment (briandowns) [reply docs/quickstart.md:65]: server should suffice.

**Review by briandowns — APPROVED** [review]

**Review by ShylajaDevadiga — APPROVED** [review]
LGTM

**Pull request merged**